Tasks/AzureRmWebAppDeployment/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -86,5 +86,10 @@
   "loc.messages.WebAppDoesntExist": "Webapp '%s' doesn't exist. Webapp should exist before deployment.",
   "loc.messages.EncodeNotSupported": "Detected file encoding of the file %s as %s. Variable substitution and Transformation is not supported with file encoding %s. Supported encodings are UTF-8 and UTF-16 LE.",
   "loc.messages.UnknownFileEncodeError": "Unable to detect encoding of the file %s (typeCode: %s). Supported encodings are UTF-8 and UTF-16 LE.",
-  "loc.messages.ShortFileBufferError": "File buffer is too short to detect encoding type : %s"
+  "loc.messages.ShortFileBufferError": "File buffer is too short to detect encoding type : %s",
```

Comment:
merge your changes with latest master and update the PR or make your changes on top of master that would be better.